quinn/src/endpoint.rs

```diff
@@ -47,6 +47,14 @@ pub struct Endpoint {
     runtime: Arc<dyn Runtime>,
 }
 
+pub struct EndpointStats {
```

Comment:
To avoid risk of overflow on long-running 32-bit applications and for consistency with `ConnectionStats`, counters should be `u64`.